buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/measurement/coverage/GenerateMeasurementsTask.groovy

```diff
@@ -54,22 +54,25 @@ public class GenerateMeasurementsTask extends DefaultTask {
         def path = "${_project.jacoco.reportsDir}/${coverageTaskName}/${coverageTaskName}.xml"
         try {
             def report = parser.parse(path)
-            def name = report.@name
             def lineCoverage = report.counter.find { it.@type == 'LINE' }
             if (lineCoverage) {
                 def covered = Double.parseDouble(lineCoverage.@covered.text())
                 def missed = Double.parseDouble(lineCoverage.@missed.text())
                 def percent = covered / (covered + missed)
-                return new Tuple(name, percent)
+                return percent
             } else {
                 throw new IllegalStateException("Cannot find line coverage section in the report: $path.")
             }
         } catch (FileNotFoundException e) {
             project.logger.warn("Cannot find coverage report for project: $_project.")
-            return new Tuple(_project.name, 0)
+            return 0
```

Comment:
Should we treat this as an error?

Reply:
Maybe.

I guess:
* if it is a problem with the coverage code itself, then it makes sense to throw an exception here;
* if it is due to an incorrectly configured project, then the code should just omit it and run through.

I haven't met a case in real life where this code path got hit, so as of now I don't know.

Reply:
What happens if the tests fail? Is there a coverage report? Or is that skipped due to the tests terminating early?

Reply:
The report generation task depends on the unit test task. Should the unit test fail, gradle will stop early and won't get to the coverage report task.

Reply:
Right, that makes sense.